### ISSUE TITLE
instrument (.ITI or .XI) loading broken ?

### ISSUE THREAD

**Issue by fabbricius** [schismtracker/schismtracker#215]
Hello,

I cannot load Fasttracker2 instruments (XI) or Impulse Tracker instruments (ITI). I use the latest stable build (20190801) on windows. I open schism, do ctrl-F4 to open the instrument library. I can browse the instruments as expected, and the samples contained in the instrument are displayed but they cannot be played (usually blank or white noise).
I have very few ITI instruments, but a lot of XI, and I wanted to used them with schism. If you want to try there are two .iti files on https://sagamusix.de/en/samples/ look for the TR-808 and TR-909 at the end of the page.
Thank you for making schism tracker, I love the interface!

**Comment by fabbricius** [comment]
edit: In fact it's impossible to even see the samples contained in an instrument with ctrl-f4. It does not even show the samples contained. The only way to see the samples is to open the sample library (using ctrl-f3) and point to a IT or XI file. But the samples are not played (whitenoise or nothing).

**Comment by jangler** [comment]
I'm not sure what  the purpose of the Instrument Library is; the way to load instruments is to go to the F4 screen and press Enter while the left pane is focused.

**Comment by fabbricius** [comment]
ok, thank you for the response. I guess I was using schismtracker wrong.

If I do F3 and go to the left pane (just at the left of "Play") and do enter, I can browse files and I can see all the files containing directly a sample (.ITS for instance) or files that embeds samples (.IT or .S3M or .XM). In that case I need to do enter on the module file to access the list of samples it contains. If I point to a sample (direct sample such as ITS or inside a module), I can see the graphical representation of the wave form, which is convenient. I can also play it using a keyboard key.

If I want instruments, .XI for instance, which are a bit easier to find than native IT instruments (.ITI), I need to type F4, then I can browse the instruments that are available in the current folder. To do that I type F4 then head to a .XI file. Schismtracker ask me if I want to enable instrument mode, and I say yes, then I can hear the instrument.

I think the .ITI file I mentionned in my first post must be broken. I cannot be sure, it it the only .ITI file I found. The problem with this ITI instument is that if I open it as a sample, it opens but the wav representation is completely mangled and it sounds as rubbish.

So no problem at the end, thanks for the support!
